pkg/api/v1beta1/types.go

```diff
@@ -750,6 +752,35 @@ type Service struct {
 
 	// Optional: Supports "ClientIP" and "None".  Used to maintain session affinity.
 	SessionAffinity AffinityType `json:"sessionAffinity,omitempty" description:"enable client IP based session affinity; must be ClientIP or None; defaults to None"`
+
+	// Optional: Ports to expose on the service.  If this field is
```

Comment:
FWIW, the long source comment isn't discoverable by users. I'd prefer to have any relevant details in the description.